app/src/main/java/org/simple/clinic/user/User.kt

```diff
@@ -138,6 +143,53 @@ data class User(
     UNAUTHORIZED;
 
     class RoomTypeConverter : RoomEnumTypeConverter<LoggedInStatus>(LoggedInStatus::class.java)
+
+  }
+
+  @Parcelize
+  data class Capabilities(val canTeleconsult: CapabilityType) : Parcelable
+
+  sealed class CapabilityType : Parcelable {
```

Comment:
Can we change the name to be more appropriate of the use case, please? 

`CapabilityType` seems like it will have different capabilities, but in fact, we are just setting the enable/disable status of the capabilities. WDYT?

Reply:
Should we name it as `Capability`?

Reply:
Probably something like `CapabilityStatus`?